generators/server/templates/src/main/java/package/config/_DefaultProfileUtil.java

```diff
@@ -20,14 +18,7 @@
      * @param app the spring application
      */
     public static void addDefaultProfile(SpringApplication app) {
-        Map<String, Object> defProperties =  new HashMap<>();
-        /*
-        * The default profile to use when no other profiles are defined
-        * This cannot be set in the <code>application.yml</code> file.
-        * See https://github.com/spring-projects/spring-boot/issues/1219
-        */
-        defProperties.put(SPRING_PROFILE_DEFAULT, Constants.SPRING_PROFILE_DEVELOPMENT);
-        app.setDefaultProperties(defProperties);
+        app.setAdditionalProfiles(Constants.SPRING_PROFILE_DEVELOPMENT, Constants.SPRING_PROFILE_SWAGGER);
```

Comment:
OK this whole spring.profiles.active and spring.profiles.include is new and not very well documented, but it seems like the way to go. I'm trying that and I'll do a new PR, as it's totally different.